addons/mail/static/tests/discuss/typing/typing.test.js

```diff
@@ -347,3 +347,38 @@ test("chat: correspondent is typing in chat window", async () => {
     );
     await contains("[title='Demo is typing...']", { count: 0 });
 });
+
+test("show typing status of members in member list", async () => {
+    const pyEnv = await startServer();
+    const userId = pyEnv["res.users"].create({ name: "Other 10" });
+    const partnerId = pyEnv["res.partner"].create({ name: "Other 10", user_ids: [userId] });
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "channel",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ partner_id: partnerId }),
+        ],
+    });
+    await start();
+    await openDiscuss(channelId);
+    await click(".o-mail-Discuss-header button[name='member-list']");
```

Comment:
Avoid picking `name=''` in test, as this is an attribute that the end-user doesn't see (unless opening dev tools but it's cheating).

Prefer `title='Show Member list'`, which is actually what the end-user sees.

Also `.o-mail-Discuss-header` is useless, and shorter selectors are preferred for improved readability.